src/components/Pages/_EventList.scss

```diff
@@ -0,0 +1,239 @@
+  }
+}
+.coa-EventListPage__EntryDetails {
+  margin-left: 24px;
+  width: 233px;
+  @include media($coa-medium-screen){
+    width: 637px;
+  }
+}
+
+.coa-EventListPage__Time {
+  font-size: 16px;
+  color: #212121;
+  font-weight: $coa-font-bold;
+  line-height: 18px;
+}
+
+.coa-EventListPage__Title {
+  font-size: 18px;
+  padding-top: 4px;
+  padding-bottom: 4px;
+  max-width: 233px;
+  line-height: 26px;
+  font-weight: 500;
+  @include media($coa-medium-screen) {
+    font-size: 24px;
+    padding-top: 8px;
+    padding-bottom: 8px;
+    max-width: 637px;
+  }
+}
+
+.coa-EventListPage__Location{
+  font-size: 16px;
+  line-height: 24px;
+  color: #5C5C5C;
+}
+
+.coa-EventListPage__Cost {
+  font-size: 16px;
+  line-height: 24px;
+  color: #5C5C5C;
+}
+
+/* Pagination */
+
+.coa-EventListPage_pagination-container {
+  margin: 40px 10px 40px;
+  /*border-top: 1px solid $coa-color-gray-lighter;*/
+  padding-top: 15px;
+  width: 740px;
+  text-align: center;
+  :first-child {
+    padding-left: 0px;
+    margin-left: 0px;
+  }
+  @include media($coa-medium-screen) {
+    width: 859px;
+  }
+}
+
+.coa-EventListPage_page {
+  color: #0049C7;
+  background-color: #F0F0F0;
+  height: 48px;
+  min-width: 48px;
+  padding: 5px;
+  margin-left: -1px;
+  font-size: 1.6rem;
+  line-height: 3.2rem;
+  font-weight: 500;
+  text-align: center;
+  cursor: pointer;
+  display: inline-block;
+  -webkit-user-select: none; /* Safari 3.1+ */
+  -moz-user-select: none; /* Firefox 2+ */
+  -ms-user-select: none; /* IE 10+ */
+  user-select: none; /* Standard syntax */
+}
+
+.coa-EventListPage_page.previous {
+  border-radius: 4px 0px 0px 4px;
+  border: none;
+}
+
+.coa-EventListPage_page.next {
+  border-radius: 0px 4px 4px 0px;
+  border: none;
+}
+
+.coa-EventListPage_page.number-container {
+  background-color: rgba(0,0,0,0);
+  color: #212121;
+  border: solid #F0F0F0 1px;
+  position: relative;
+  height: 48px;
+  top: -1px;
+  @include mobile {
+    width: 38px;
+    min-width: 38px;
+  }
+}
+
+.coa-EventListPage_page.ellipsis {
+  border: none;
+  pointer-events: none;
+}
+
+
+.coa-EventListPage_number {
+  font-weight: 900;
+  position: relative;
+  top: 2px;
+}
+
+.coa-EventListPage_page.number-container.active {
+  color: #0049C7;
+  background-color: #D9E8F6;
+  pointer-events: none;
+}
+
+.coa-EventListPage_page-chevron {
+  position: relative;
+  height: 24px;
+  top: 7px;
+  left: 1px;
+}
+
+.coa-EventListPage_text {
+  display: inline-block;
+  padding-left: 11px;
+  padding-right: 10px;
+  height: 32px;
+  line-height: 30px;
+  border-left: 1px solid #dbdee1;
+}
+
+.coa-EventListPage_text.right {
+  border: none;
+  border-right: 1px solid #dbdee1;
+  padding-left: 8px;
+  padding-right: 12px;
+}
+
+.coa-EventListPage_page:hover {
+  background-color: #0049C7;
+  color: #FFFFFF;
+}
+
+.coa-EventListPage_page.active:hover {
+  background-color: #0049C7;
+  color: #FFFFFF;
+}
+
+.coa-EventListPage_page:hover .coa-CheveronLeftBlue {
```

Comment:
definitely not something that should stop us from shipping, but it sparked my scss brain a bit and i think this could be done a little more sassily, something along the lines of:
```
.coa-EventsListPage_page {
  &:hover {
    .coa-ChevronLeftBlue {
      fill: #FFFFFF;
    }
  }
}
```